config/config_test.go

```diff
@@ -679,8 +679,8 @@ var expectedConf = &Config{
 			HTTPClientConfig: config.DefaultHTTPClientConfig,
 
 			ServiceDiscoveryConfigs: discovery.Configs{
-				&dockerswarm.SDConfig{
-					Filters:          []dockerswarm.Filter{},
+				&moby.SDConfig{
```

Comment:
As there is now `moby.DockerSDConfig`, does it maybe make more sense to call this one `moby.SwarmSDConfig`? I had thought about this in my PR already but wanted to make the change "as small as possible".

Reply:
I have renamed, can you review again?